src/core/MRApp.js

```diff
@@ -410,7 +424,37 @@ export class MRApp extends MRElement {
             this.stats.end();
         }
 
-        this.renderer.render(this.scene, this.user);
+        // ----- Actually Render ----- //
+
+        // TODO (in future) - once this gets more complicated, it will be nice to have a render system separate
+        // from the pure loop but it is okay as is here for now.
+
+        // Need to wait until we have all needed rendering-associated systems loaded.
+        if (this.maskingSystem == undefined) {
+            return;
+        }
+
+        const camera = this.user;
```

Comment:
Let's just pass `this.user` into the render calls.

Reply:
sounds good ~